federatedscope/core/configs/cfg_fl_algo.py

```diff
@@ -63,6 +63,13 @@ def extend_fl_algo_cfg(cfg):
     cfg.personalization.K = 5  # the local approximation steps for pFedMe
     cfg.personalization.beta = 1.0  # the average moving parameter for pFedMe
 
+    # parameters for FedRep：
```

Comment:
Please add explanations on those new configs; Please also the explanation of those new configs to the file FederatedScope/federatedscope/core/configs/README.md